From 63ba5e2d60fc01c0e82287a1459e68f999859bea Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Thu, 3 Jun 2021 17:51:03 +0200
Subject: [PATCH] Fix parallel build issue

Tell meson that the generated gdbofono headers are needed at compile
time.

fixes https://gitlab.com/mobian1/devices/eg25-manager/-/issues/14
---
 src/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/meson.build b/src/meson.build
index f9eb27f..1bfcd3a 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -15,6 +15,7 @@ src = [
         'suspend.c', 'suspend.h',
         'toml.c', 'toml.h',
         'udev.c', 'udev.h',
+        gdbofono_headers,
 ]
 
 if mmglib_dep.found()
-- 
GitLab

